front/app/components/admin/TabbedResource/index.tsx

```diff
@@ -103,116 +62,52 @@ const ChildWrapper = styled.div`
   }
 `;
 
-const StatusLabelWithMargin = styled(StatusLabel)`
-  margin-left: 12px;
-`;
-
-type Props = {
+interface Props {
   resource: {
     title: string;
     subtitle?: string;
   };
-  tabs?: ITab[];
-  children?: React.ReactNode;
-};
-
-interface State {}
-
-function getRegularExpression(tabUrl: string) {
-  return new RegExp(`^/([a-zA-Z]{2,3}(-[a-zA-Z]{2,3})?)(${tabUrl})(/)?$`);
+  tabs: ITab[];
+  children: React.ReactNode;
```

Comment:
Also `children` should be required. You won't use TabbedResource without child content.